website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/connecting-your-database.md

```diff
@@ -111,3 +113,15 @@ In addition to these fields, there are two other optional fields that can be con
 
 
 <Lightbox src="/img/docs/dbt-cloud/cloud-configuring-dbt-cloud/bigquery-connection.png" title="A valid BigQuery connection"/>
+
+### BigQuery OAuth
+**Available in:** Development environments, Enterprise plans only
+
+The OAuth auth method permits dbt Cloud to run development queries on behalf of
+a BigQuery user without the configuration of BigQuery password in dbt Cloud. For
+more information on the initial configuration of a BigQuery OAuth connection in dbt Cloud, please see
+[the docs on setting up BigQuery OAuth](cloud-setting-up-bigquery-oauth).
+
+As an end user, you can connect via OAuth by clicking the Edit button in your BigQuery Connection page, and adding your Client ID and Secret for the connection. You'll need that Client ID and Secret specifically from BigQuery to set up this connection - discuss with your database administrator how to go about getting those for your organization.
```

Comment:
Whoa! I don't think this is accurate - end-users don't need to specify their own client IDs and client secrets! I think the thing to say here is that end-users can integrate their Cloud user with their BigQuery user by navigating to their dbt Cloud Profile page, selecting the relevant project in the "Credentials" nav, and then clicking the button to log in with BigQuery!

Reply:
Yeah, this was a misunderstanding on my side as to how Enterprise organizations would be navigating this setup!  Fixed in the incoming commit